test/index.js

```diff
@@ -4,18 +4,11 @@
 require('source-map-support').install();
 
 global.sinon = require('sinon');
+require('./chai-setup');
+global.chai.use(require('sinon-chai'));
 
-const chai = (global.chai = require('chai'));
-
-chai.use(require('sinon-chai'));
-chai.should();
-
-const bluebird = require('bluebird');
-global.expect = chai.expect;
 global.d = new Date();
```

Comment:
Hmm... having a global variable named `d` seems a bit dangerous since `d` is commonly used as a temporary variable name.  This could lead to some surprises when writing test cases.

(Fixing this might be out of scope for the current PR, though.  But, if we can fix this in 1-2 lines, then it might be worth it).

Reply:
note: usages can be found via search /* global.?d */